libs/spark/src/Unstable_Button/Unstable_Button.tsx

```diff
@@ -79,13 +79,20 @@ const buttonFontVariantLarge = buildVariant(
 const useStyles = makeStyles<Unstable_ButtonClassKey>(
   (theme) => ({
     root: (props: Unstable_ButtonProps) => ({
-      borderColor: 'transparent',
+      // Double specificity to override V1 theme style
+      '&&': {
+        borderColor: 'transparent',
+      },
       borderRadius: 4,
       borderStyle: 'solid',
       borderWidth: 1,
       '&.Mui-focusVisible, &:focus-visible': {
         boxShadow: `0px 0px 2px 4px ${theme.unstable_palette.teal[300]}`,
       },
+      // Double specificity to override V1 theme style
+      '&&:disabled': {
+        opacity: 'unset',
+      },
```

Comment:
```suggestion
      // override V1 theme style
      '&&:disabled': {
        opacity: 'unset',
      },
```
It's only to override the style. Not an existing style that needs double specificity.